gatsby-node.js

```diff
@@ -16,22 +16,6 @@ exports.onCreatePage = ({ page, actions }) => {
     const is_story = /story/g.test(page.path)
     const is_market = /markets/g.test(page.path)
     const is_cfds = /cfds/g.test(page.path)
-    const is_landing_ebooks =/landing\/ebooks/g.test(page.path)
-    
-    if (is_landing_ebooks) {
```

Comment:
Why are we removing thispiece of code?

Reply:
because it's for redirection from /landing/ebooks (to fix 403 error and redirect to 404) But then this redirection was made inside deriv config for all 403 pages to 404 page